docs/src/introduction.md.ejs

```diff
@@ -1,5 +1,11 @@
 # Introduction
 
-RippleAPI allows you to query and submit transactions to a node on the Ripple network.
+RippleAPI is the official client library to the Ripple Consensus Ledger. Currently, RippleAPI is only available in JavaScript. 
+Using RippleAPI, you can:
+
+* [Query transactions from the network](#gettransaction)
+* [Sign](#sign) transactions securely without connecting to any server
+* [Submit](#submit) transactions to the Ripple Consensus Ledger, including [Payments](#payment), [Orders](#order), [Settings changes](#settings), and [other types](#transaction-types)
+* [Generate a new Ripple Address](#generateaddress)
```

Comment:
We should say "and more" or something so that it doesn't sound like this is all that it can do.